### PR TITLE
fix broken tests

### PR DESCRIPTION
**Pull request by dalfrank**

### CODE REVIEW

File: def/proto1/config/environments/development.rb [hunk 1]
```diff
@@ -22,7 +22,7 @@
   config.action_mailer.raise_delivery_errors = true
 
   # Directory for table backups made by the data controller
-  DATA_CONTROLLER_BACKUP_DIR = '/proj/defExtra/dataTableBackups'
+  DATA_CONTROLLER_BACKUP_DIR = 'TBD, e.g. /proj/defExtra/dataTableBackups'
```
Comment (plynchnlm) [inline def/proto1/config/environments/development.rb:25]: Is there anything in the installation instructions about the need to set these to real paths?

**Review by plynchnlm — CHANGES_REQUESTED** [review]

File: def/proto1/config/environments/development.rb (hunk 1 above)
Comment (dalfrank) [reply def/proto1/config/environments/development.rb:25]: It's been mentioned here as "Elements Flagged for Change" https://lhncbc.github.io/phr/configuration.html. Looks like we also need to include development.rb, test.rb, and production.rb in the file list (see pull request https://github.com/lhncbc/phr/pull/8)

Comment (plynchnlm) [reply def/proto1/config/environments/development.rb:25]: Okay, but that file does not mention this one (development.rb).  Please add this file to the list there.

Comment (plynchnlm) [reply def/proto1/config/environments/development.rb:25]: Oh-- I see that is in your other pull request.  Okay.

**Review by plynchnlm — APPROVED** [review]

**Pull request merged**